sklearn/manifold/t_sne.py

```diff
@@ -142,10 +142,12 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     n = pdist(X_embedded, "sqeuclidean")
-    n += 1.
```

Comment:
Let me know if you would like me to post an explanation of KL - correction, I have done the calculus to confirm -- although you can tell that my code works simply my running the program if you wish! (make sure to pass method='exact')